test/baselines/adapter_JS_ES6Class_ESLint_TypeChecking_Spaces_SingleQuotes_Apache-2.0/__meta__/npm_package_files.txt

```diff
@@ -15,7 +15,7 @@ name:          iobroker.test-adapter
 version:       0.0.1                                   
 filename:      iobroker.test-adapter-0.0.1.tgz         
 package size:  79.5 kB                                 
-unpacked size: 112.3 kB                                
-shasum:        c40c16b23db623e79e0845e0f170b1f004f93a19
```

Comment:
OK, this is getting ugly: we should only keep the file names (even without size) in this file, otherwise the baseline diffs will be unreadable and we will not spot the important differences.

Reply:
Simplified